### ISSUE TITLE
Show duration and cost if available on Category:Self-Report Measure

### ISSUE THREAD

**Issue by dgfitch** [uwmadison-chm/factuator#6]
If the self reports have a cost and/or duration and/or number of items, display those somehow conveniently on the top level listing by construct.

Like:

PANAS-NOW (Free, 10 minutes, 7 items)

Also, it would be good to alter the display so that the headers don't split their subitems. This may require using different CSS or altering the base CSS?

**Comment by eguzman-wisc** [comment]
Hey Dan, not sure what you mean here by "it would be good to alter the display so that the headers don't split their subitems". Got any more info on what you would like to be done?

**Comment by dgfitch** [comment]
Yeah, it's more of a CSS/display thing... it happens both on the construct listing and on the study "by status" listing because we're using the mediawiki "3 column" CSS.

![image](https://user-images.githubusercontent.com/2800/58028528-8b4f3e00-7ae0-11e9-9106-07e9043de3b7.png)

See how the header splits from the items after it?
